src/com/amazon/corretto/crypto/provider/EvpSignatureRaw.java

```diff
@@ -7,75 +7,62 @@
 import java.security.SignatureException;
 
 class EvpSignatureRaw extends EvpSignatureBase {
-    private final AmazonCorrettoCryptoProvider provider_;
-    private AccessibleByteArrayOutputStream buffer = new AccessibleByteArrayOutputStream(64, 1024*1024);
+    private AccessibleByteArrayOutputStream buffer = new AccessibleByteArrayOutputStream(64, 1024 * 1024);
 
     private EvpSignatureRaw(AmazonCorrettoCryptoProvider provider, final EvpKeyType keyType, final int paddingType) {
-        super(keyType, paddingType);
-        provider_ = provider;
+        super(provider, keyType, paddingType);
     }
 
-    @Override protected void engineReset() {
+    @Override
+    protected void engineReset() {
         buffer.reset();
     }
 
-    @Override protected void engineUpdate(final byte b) throws SignatureException {
+    @Override
+    protected void engineUpdate(final byte b) throws SignatureException {
         buffer.write(b & 0xFF);
```

Comment:
i see. seems like an odd API contract decision [on the part of OutputStream](https://docs.oracle.com/javase/8/docs/api/java/io/OutputStream.html#write-int-). FWIW, OutputStream just ignores the higher-order bits irrelevant to a byte value:

> The byte to be written is the eight low-order bits of the argument b. The 24 high-order bits of b are ignored.

outside scope of this review, but it might be better to do this `& 0xff` masking in `AccessibleByteArrayOutputStream.write` so callers don't have to worry about it (or silently introduce a bug by neglecting mask).